### DISCUSSION TITLE
TabControl中用UserControl内容无法填满的问题

### DISCUSSION THREAD

**Discussion by MagicGunner** [BYJRK/DotNet-Discussions#190] [WPF 开发]
<img width="960" alt="微信图片_20231223103521" src="https://github.com/BYJRK/DotNet-Discussions/assets/38397610/de065afa-fa40-4ce4-92a3-c8a7b38c4890">
大佬您好，WPF新手求教关于布局的问题

**Comment by xingzuhui** [comment]
和我的问题一样
